### ISSUE TITLE
IA: External audience interviews - final steps 

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4484]
Blocked by: #4483
- [ ] do the interviews
- [ ] compile findings and compare them with Internal Stakeholders Interviews
- [ ] adjust wireframes

**Comment by taisdesouzalessa** [comment]
Since we have an Audience working group in 2021 this ticket is no longer relevant. The working group will come up with a process to gather insights about our target audience. Closing this ticket.
